community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/codegen/ir/ProjectionInstruction.scala

```diff
@@ -24,46 +24,55 @@ import org.neo4j.cypher.internal.compiler.v2_3.codegen.JavaUtils.JavaTypes.{DOUB
 import org.neo4j.cypher.internal.compiler.v2_3.codegen.{CodeGenerator, KernelExceptionCodeGen, Namer}
 
 sealed trait ProjectionInstruction extends Instruction {
+
   def projectedVariable: JavaSymbol
+
   def generateCode() = ""
-  override protected def _importedClasses() = Set.empty
 }
 
 object ProjectionInstruction {
+
   def literal(value: Long): ProjectionInstruction = ProjectLiteral(JavaSymbol(value.toString + "L", LONG))
+
   def literal(value: Double): ProjectionInstruction = ProjectLiteral(JavaSymbol(value.toString, DOUBLE))
-  def literal(value: String): ProjectionInstruction = ProjectLiteral(JavaSymbol( s""""$value"""", STRING))
+
+  def literal(value: String): ProjectionInstruction = ProjectLiteral(JavaSymbol(s""""$value"""", STRING))
+
   def literal(value: Boolean): ProjectionInstruction = ???
```

Comment:
I know this is not changed here but why do we have this here? it should either be removed or return `ProjectLiteral(JavaSymbol(value.toString, BOOLEAN))`